logger/logging_config.py

```diff
@@ -0,0 +1,18 @@
+import json
+from logging.config import dictConfig  # type: ignore
+
+done_setup = False
+
+
+def setup_logging():
+    global done_setup
+
+    if not done_setup:
+        try:
+            logging_config_file_path = 'logger/logging_config.json'
+            with open(logging_config_file_path, 'rt') as file:
+                config = json.load(file)
+            dictConfig(config)
```

Comment:
I'm a bit confused by this working without that indentation, with must not use scope other than what its working on. Fixed anyway